deploy/kubernetes/Makefile

```diff
@@ -1,4 +1,4 @@
-ALL=filebeat metricbeat auditbeat heartbeat elastic-agent-standalone
+ALL=filebeat metricbeat auditbeat heartbeat elastic-agent-standalone elastic-agent-managed
```

Comment:
@ruflin do you think naming is ok here? Should we call it just `elastic-agent` maybe? Or `elastic-agent-fleet`? `elastic-agent-managed` is fine for me too.

Reply:
No strong preference. `managed` looks fine as we have also `standalone`.